egs/wsj/s5/steps/libs/nnet3/train/frame_level_objf/common.py

```diff
@@ -389,38 +424,63 @@ def compute_train_cv_probabilities(dir, iter, egs_dir, left_context,
     context_opts = "--left-context={lc} --right-context={rc}".format(
         lc=left_context, rc=right_context)
 
+    if use_multitask_egs:
+        valid_diagnostic_egs = "scp:{0}/valid_diagnostic.egs.1.scp".format(
```

Comment:
I used same allocate_multilingual_example.py for generating egs.archive_index.scp for training and valid and num_archives is 1 for valid set.